client/src/__tests__/inspect.test.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event'
+import { navigate } from '@reach/router';
+import '@testing-library/jest-dom/extend-expect'
+import Inspect from '../pages/Inspect';
+import { deleteTransaction, getOneTransaction, getTransactions, editTransaction } from '../services/TransactionService';
+
+jest.mock('../services/TransactionService');
+getOneTransaction.mockResolvedValue('upload');
+getTransactions.mockResolvedValue('upload');
+deleteTransaction.mockResolvedValue('upload');
+editTransaction.mockResolvedValue('upload');
+
+jest.mock('@reach/router')
+navigate.mockResolvedValue('');
+
+const users =
+{
+  _id: "5f75e077dd559746a01dda55",
+  lead: "C",
+  leadEmail: "1234@test",
+  partner: "M",
+  partnerEmail: "123@test"
+};
+const currency = '£';
+// _id
```

Comment:
//ehh commented code
Though i've probably left a bunch of commented code in my pull requests too